### DISCUSSION TITLE
[Esame 9 Gennaio 2024] Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#428] [Esami - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/a769c567-f086-4f54-bb40-0ee1b60f132c)

**Comment by Elia-Belli** [comment, marked as answer]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/fa69e58e-8d83-4132-845b-231bd7f2e0aa)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/60f52ef5-4196-479d-9117-56e659e00034)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-30 152920](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/b12334d7-439c-4195-ad62-70f1a2fddcc0)
![Immagine 2024-01-30 152939](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/94fc9332-e711-484a-9c36-f4d36b1ac15a)
![Immagine 2024-01-30 152956](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/de377309-d79f-4f9b-ae9e-f5562d25a95b)
![Immagine 2024-01-30 153009](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/3786b2d9-d616-46f1-815e-5420dcd8f45a)
